docs/api/ratsd.yaml

```diff
@@ -0,0 +1,148 @@
+openapi: 3.0.0
+info:
+  title: RATS Evidence Collection Daemon
+  version: 0.0.1
+tags: []
+paths:
+  /ratsd/chares:
+    post:
+      operationId: Ratsd_chares
```

Comment:
What is chares other than the mythical ferryman? I’m trying to understand if 401 or 403 is the most appropriate error code

Reply:
At least for now, it seems sufficient to be successfully authenticated to request evidence using this challenge-response API.  I cannot imagine a case where users who have provided an authentication token that checks out aren't also allowed to query this interface.  In the future, we may add a 403 to handle such cases but for now 401 looks appropriate to me.